libkbfs/folder_block_ops.go

```diff
@@ -960,7 +960,7 @@ func (fbo *folderBlockOps) nowUnixNano() int64 {
 // be the new DirEntry, and a local block cache. It also modifies md,
 // which must be a copy.
```

Comment:
Which part, modifying `md`? It is -- there's a call to `md.AddOp()`, which I missed last time.